graphrag/query/context_builder/local_context.py

```diff
@@ -155,7 +155,7 @@ def build_covariates_context(
 def build_relationship_context(
     selected_entities: list[Entity],
     relationships: list[Relationship],
-    token_encoder: tiktoken.Encoding | None = None,
+    tokenizer: Tokenizer = get_tokenizer(),
```

Comment:
it is usually bad practice to set a default as a function, this causes that once the function is loaded (the file imported or the function itself) it will set that tokenizer and not a default that gets created every time it runs, it could be fine for this function in particular, but wanted to point this out

Reply:
Moved instantiation/fallback into the method.